scripts/spec_xml/generate_spec_xml.py

```diff
@@ -24,26 +24,17 @@
 from pathlib import Path
 
 import click
+from paths import Branch, get_chip_root, get_data_model_path, get_documentation_file_path, get_in_progress_defines
 
-DEFAULT_CHIP_ROOT = os.path.abspath(
-    os.path.join(os.path.dirname(__file__), '..', '..'))
-DEFAULT_OUTPUT_DIR_1_3 = os.path.abspath(
-    os.path.join(DEFAULT_CHIP_ROOT, 'data_model', '1.3'))
-DEFAULT_OUTPUT_DIR_IN_PROGRESS = os.path.abspath(
-    os.path.join(DEFAULT_CHIP_ROOT, 'data_model', 'in_progress'))
-DEFAULT_OUTPUT_DIR_TOT = os.path.abspath(
-    os.path.join(DEFAULT_CHIP_ROOT, 'data_model', 'master'))
-DEFAULT_DOCUMENTATION_FILE = os.path.abspath(
-    os.path.join(DEFAULT_CHIP_ROOT, 'docs', 'spec_clusters.md'))
-
-# questions
-# is energy-calendar still in?
-# is heat-pump out? wasn't in 0.7
-# location-cluster - is this define gone now?
-# queuedpreset - is this define gone now?
-CURRENT_IN_PROGRESS_DEFINES = ['aliro', 'atomicwrites', 'battery-storage', 'device-location', 'e2e-jf', 'energy-calendar', 'energy-drlc',
-                               'energy-management', 'heat-pump', 'hrap-1', 'hvac', 'matter-fabric-synchronization', 'metering', 'secondary-net',
-                               'service-area-cluster', 'solar-power', 'tcp', 'water-heater', 'wifiSetup']
+# Use the get_in_progress_defines() function to fetch the in-progress defines
+CURRENT_IN_PROGRESS_DEFINES = get_in_progress_defines()
+
+# Replace hardcoded paths with dynamic paths using paths.py functions
+DEFAULT_CHIP_ROOT = get_chip_root()
+DEFAULT_OUTPUT_DIR_1_3 = get_data_model_path(Branch.V1_3)
+DEFAULT_OUTPUT_DIR_IN_PROGRESS = get_data_model_path(Branch.MASTER)
```

Comment:
in Previous code, IN_PROGRESS dir went to "in_progress" and TOT went to "master".

In the updated code, both go to Master. why?